subsys/net/openthread/rpc/server/ot_rpc_message.c

```diff
@@ -24,66 +24,63 @@ OT_RPC_RESOURCE_TABLE_REGISTER(msg, otMessage, CONFIG_OPENTHREAD_RPC_MESSAGE_POO
 static void ot_rpc_msg_free(const struct nrf_rpc_group *group, struct nrf_rpc_cbor_ctx *ctx,
 			    void *handler_data)
 {
-	ot_rpc_res_tab_key key = 0;
+	ot_rpc_res_tab_key key;
```

Comment:
why remove assignment?

Reply:
because it's assigned a few lines below so the initialization here is pointless.